config/sync/field.field.block_content.featured_game.field_background_image.yml

```diff
@@ -0,0 +1,43 @@
+uuid: 4e9341fd-f651-4c4e-bfc9-e96f17741b38
+langcode: en
+status: true
+dependencies:
+  config:
+    - block_content.type.featured_game
+    - field.storage.block_content.field_background_image
+  module:
+    - bg_img_field
+id: block_content.featured_game.field_background_image
+field_name: field_background_image
+entity_type: block_content
+bundle: featured_game
+label: 'background image'
+description: ''
+required: false
+translatable: true
+default_value: {  }
+default_value_callback: ''
+settings:
+  css_settings:
+    css_selector: ''
+    css_repeat: inherit
+    css_background_size: inherit
+    css_background_position: inherit
```

Comment:
How is this working? :) Shouldn't you add a selector here?
Also, are the images responsive?
Consider using the following values:
```
css_repeat: no-repeat
css_background_size: cover
css_background_position: center (or according to your needs)
```
https://www.w3schools.com/cssref/pr_background-repeat.asp
https://www.w3schools.com/cssref/css3_pr_background-size.asp
https://www.w3schools.com/cssref/pr_background-position.asp